.github/workflows/tests.yml

```diff
@@ -53,6 +53,35 @@ jobs:
         with:
           name: bazel-testlogs
           path: bazel-testlogs
+
+  build_bazel4_arm64_simulator:
+    name: Build arm64 Simulator
```

Comment:
Nit: So we don't have a `bazel4_arm64_simulator_arm64_host` test too because atm we can't run on such hosts on CI right? Should this name explicitly mention `x86_64` host?

Reply:
Yeah exactly, you'd need an Apple Silicon CI!